### DISCUSSION TITLE
When we put human ethics and value judgments into AI, let it understand all human behaviors and emotions, let it communicate with human beings in a natural and emotional way, and at the same time, make it wholeheartedly serve human beings, and can sacrifice itself at any time. What is it? What is the difference between it and a dog? Is a dog safe? Is a human level dog safe?

### DISCUSSION THREAD

**Discussion by lets-make-safe-ai** [lets-make-safe-ai/make-safe-ai#15] [General]
When we put human ethics and value judgments into AI, let it understand all human behaviors and emotions, let it communicate with human beings in a natural and emotional way, and at the same time, make it wholeheartedly serve human beings, and can sacrifice itself at any time. What is it? What is the difference between it and a dog? Is a dog safe? Is a human level dog safe?

**Comment by chirag127** [comment]
can you please move these to different discussions, too many questions.
